data/src/main/java/org/oppia/data/backends/gae/NetworkModule.kt

```diff
@@ -0,0 +1,68 @@
+package org.oppia.data.backends.gae
+
+import dagger.Module
+import dagger.Provides
+import okhttp3.OkHttpClient
+import org.oppia.data.backends.gae.api.ClassroomService
+import org.oppia.data.backends.gae.api.TopicService
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+/**
+ * Sample resource https://github.com/gahfy/Feed-Me/tree/unitTests
```

Comment:
Thanks--do you want to present this in one of our upcoming mobile meetings? I wanted to cover testing & Dagger directly, but this might be a nice add-on topic.

Reply:
I do think it is worth sharing with everyone. I am just not sure about finish this by Monday. Anyways, I will prepare for this and will present on Monday.